### ISSUE TITLE
E1812 unwanted pairing to LED1836G9

### ISSUE THREAD

**Issue by hank-droid** [Koenkk/zigbee2mqtt#8871]
## What happened
when I press E1812, LED1836G9 switch on
## What did you expect to happen
nothing
## How to reproduce it (minimal and precise)
pairing sequence:

1. E1812
2.  LED1836G9
3. E1524/E1810 
        then paired to LED1836G9, 
         LED1836G9 unplugged
then plugged back

second try pairing sequence:

1.LED1836G9
2.E1524/E1810 
        then paired to LED1836G9, 
        LED1836G9 unplugged
3. E1812
then LED1836G9 plugged back

## Debug info
Zigbee2MQTT version: 1.21.1-1
Adapter hardware: CC2652P
Adapter firmware version: 1.21.1-dev commit: 27a7bdb

## herdsman debug logging
2021-09-25T09:33:14.904Z zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,23,68,129,133,3,6,0,73,251,1,1,0,63,0,22,225,223,0,0,3,1,2,1,141,158,10,239]
2021-09-25T09:33:14.904Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,23,68,129,133,3,6,0,73,251,1,1,0,63,0,22,225,223,0,0,3,1,2,1,141,158,10,239]
2021-09-25T09:33:14.905Z zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 23 - 2 - 4 - 129 - [133,3,6,0,73,251,1,1,0,63,0,22,225,223,0,0,3,1,2,1,141,158,10] - 239
2021-09-25T09:33:14.906Z zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":901,"clusterid":6,"srcaddr":64329,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":63,"securityuse":0,"timestamp":14672150,"transseqnumber":0,"len":3,"data":{"type":"Buffer","data":[1,2,1]}}
2021-09-25T09:33:14.909Z zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":2,"manufacturerCode":null,"commandIdentifier":1},"Payload":{},"Command":{"ID":1,"parameters":[],"name":"on"}},"address":64329,"endpoint":1,"linkquality":63,"groupID":901}'
2021-09-25T09:33:14.912Z zigbee-herdsman:controller:endpoint DefaultResponse 0x84fd27fffeaae54c/1 6(1, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false})
2021-09-25T09:33:14.913Z zigbee-herdsman:adapter:zStack:adapter sendZclFrameToEndpointInternal 0x84fd27fffeaae54c:64329/1 (0,0,1)
2021-09-25T09:33:14.915Z zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":64329,"destendpoint":1,"srcendpoint":1,"clusterid":6,"transid":209,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,2,11,1,0]}}
2021-09-25T09:33:14.916Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,73,251,1,1,6,0,209,0,30,5,24,2,11,1,0,68]
2021-09-25T09:33:14.918Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []
Zigbee2MQTT:info  2021-09-25 11:33:14: MQTT publish: topic 'zigbee2mqtt/0x84fd27fffeaae54c', payload '{"action":"on","battery":47,"linkquality":63,"update":{"state":"idle"},"update_available":false}'
Zigbee2MQTT:info  2021-09-25 11:33:14: MQTT publish: topic 'zigbee2mqtt/0x84fd27fffeaae54c', payload '{"action":"","battery":47,"linkquality":63,"update":{"state":"idle"},"update_available":false}'
Zigbee2MQTT:info  2021-09-25 11:33:14: MQTT publish: topic 'zigbee2mqtt/0x84fd27fffeaae54c/action', payload 'on'
2021-09-25T09:33:14.945Z zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100]
2021-09-25T09:33:14.947Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100]
2021-09-25T09:33:14.947Z zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100
2021-09-25T09:33:14.948Z zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0}
2021-09-25T09:33:14.949Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []
2021-09-25T09:33:14.954Z zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,1,209,23]
2021-09-25T09:33:14.955Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,1,209,23]
2021-09-25T09:33:14.956Z zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,1,209] - 23
2021-09-25T09:33:14.958Z zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":1,"transid":209}
2021-09-25T09:33:14.958Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []
## Notes
issue occurs with zigbee2mqtt add-on stopped as well, as long as the E1812 were joined to zigbee2mqtt
it seems that as soon as E1812  join the zigbee2mqtt, then their button switches on the LED1836G9

**issue stops occurring when E1812 is removed via interface** _(no force remove, no reset of E1812)_

same happens  with the IKEA N2 remote (STYRFRY, the newest silver model) after being paired to zigbee2mqtt and then to IKEA ICPSHC24-10EU-IL-1, pressing the on/brightness up button would switch on both the LED1836G9

**Comment by hank-droid** [comment]
i am not so concerned about the E1812, but that it can occur to the N2 if it would be added to stable version, since i plan to have 10 of them in my set up and i would like them to be in the zigbee2mqtt network, rather than only paired to the bulbs to avoid this conflict.
thank you!

**Comment by hank-droid** [comment]
i run multiple tests with the ICPSHC24-10EU-IL-1 and N2 remote as well, in genera it seems that if i join them to the zigbee coordinator, then they all act as if they are part of the same group.

**Comment by BalassaMarton** [comment]
I have the exact same problem with E1743 (on/off switch), LED1842G3 (bulb) and ICPSHC24-10EU-IL-1 (10w LED driver). I even created bind groups for each so that they are not in the same group, but no success. The LED driver and bulb can be controlled separately from UI and MQTT, but not using the switch. I have multiple switches, some bound to the LED driver, others to the bulb, but they all control both lights, no matter what I do. Removing the switches from the default bind group always fails.

**Comment by BalassaMarton** [comment]
Update - I was able to remove SOME of the dimmers from the default bind group, then added them to the respective groups (one for each light), but they still switch both lights.

**Comment by BalassaMarton** [comment]
Solved! I did the below steps in this exact order (after force-removing the devices):

1. Add the lights
1. Create a dedicated group for each light
3. Put each light into its dedicated group
5. Add the first switch
6. Bind the switch to its respective group, without removing from the default bind group
7. Go to Dashboard, verify that the switch is controlling the correct light
8. Repeat from step 4 for remaining switches

Note: my switches have version 2.2.010 firmware, which appears as the latest as of today.
